sdk/README.md

```diff
@@ -81,3 +81,16 @@ Here we update the `Compiler` of the KFP SDK to generate `Tekton` YAML for a bas
 ## Test Kubeflow Pipelines with Tekton
 
 Please [refer to the instructions here](./python/tests/README.md) as you work on a PR test sample Kubeflow Pipelines in their test data folder to ensure your PR is improving the number of successful samples
+
+## Compile Kubeflow Pipelines as Tekton pipelineRun
+
+Compiling Kubeflow Pipelines into Tekton pipelineRun is currently under the experimental stage. Below are the features that only supported within pipelineRun
```

Comment:
This is a good place to add couple of details and difference between pipelinRun and static pipeline, and why pipelineRun is closer to Argo yaml

Reply:
I added reasons why we need to generate pipelinerun and pointed them to the tekton vs argo section in the design docs.